### PR TITLE
Update EUM-Server exporter documentation

### PR DESCRIPTION
**Pull request by heiko-holz**
As part of https://github.com/inspectIT/inspectit-ocelot-eum-server/issues/12, this PR updates the documentation of the tracing and metrics exporter of the EUM.

- - -
This change is [Reviewable](https://reviewable.io/reviews/inspectIT/inspectit-ocelot/1569)

### CODE REVIEW

File: inspectit-ocelot-documentation/docs/enduser-monitoring/eum-server-configuration.md [hunk 1]
```diff
@@ -466,21 +466,21 @@ inspectit-eum-server:
...
-The following configuration snippet shows the default configuration of the prometheus-exporter:
+The following configuration snippet shows the default configuration of the prometheus-exporter, exposing on port `8888`:
 ```YAML
 inspectit-eum-server:
   exporters:
     metrics:
       prometheus:
         # Determines whether the prometheus exporter is enabled.
-        enabled: true
+        enabled: ENABLED
```
Comment (PDuwe) [inline inspectit-ocelot-documentation/docs/enduser-monitoring/eum-server-configuration.md:483]: Maybe I misunderstood something, but you say earlier that by default prometheus exporter is disabled (l. 474)
This however suggests, that it is enabled. Or am I mistaken? :)

**Review by heiko-holz — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/inspectIT/inspectit-ocelot/1569)* status: 0 of 2 files reviewed, 1 unresolved discussion (waiting on @PDuwe)

___
*[`inspectit-ocelot-documentation/docs/enduser-monitoring/eum-server-configuration.md` line 483 at r1](https://reviewable.io/reviews/inspectIT/inspectit-ocelot/1569#-NMcLIJTOvLaoVSApx1h-r1-483:-NMcMXiR684Re1ZL8gZ9:b2nrjmo) ([raw file](https://github.com/inspectIT/inspectit-ocelot/blob/59b08a083489eb329eaaa492fad46e03a75e7e15/inspectit-ocelot-documentation/docs/enduser-monitoring/eum-server-configuration.md#L483)):*
<details><summary><i>Previously, PDuwe wrote…</i></summary><blockquote>

Maybe I misunderstood something, but you say earlier that by default prometheus exporter is disabled (l. 474)
This however suggests, that it is enabled. Or am I mistaken? :) 
</blockquote></details>

Hey,
yes the Prometheus exporter is disabled **by default**. 
The snippet shows how to enable it (written in lines above *The following configuration snippet shows the default configuration of the prometheus-exporter, exposing on port `8888`:*)

**Review by Dimi-Ma — DISMISSED** [review]
> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/inspectIT/inspectit-ocelot/1569)* status: all files reviewed, 1 unresolved discussion (waiting on @PDuwe)

___
*[`inspectit-ocelot-documentation/docs/enduser-monitoring/eum-server-configuration.md` line 483 at r1](https://reviewable.io/reviews/inspectIT/inspectit-ocelot/1569#-NMcLIJTOvLaoVSApx1h-r1-483:-NMcNPeeAILhp1r9okAz:b-uh5ol8) ([raw file](https://github.com/inspectIT/inspectit-ocelot/blob/59b08a083489eb329eaaa492fad46e03a75e7e15/inspectit-ocelot-documentation/docs/enduser-monitoring/eum-server-configuration.md#L483)):*
<details><summary><i>Previously, heiko-holz (Heiko Holz) wrote…</i></summary><blockquote>

Hey,
yes the Prometheus exporter is disabled **by default**. 
The snippet shows how to enable it (written in lines above *The following configuration snippet shows the default configuration of the prometheus-exporter, exposing on port `8888`:*)
</blockquote></details>

It might lead to misunderstandings if you say that this snippet shows the default configuration of the prometheus-exporter with it being enabled but say that it is disabled by default.

**Review by heiko-holz — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/inspectIT/inspectit-ocelot/1569)* status: 1 of 2 files reviewed, 1 unresolved discussion (waiting on @Dimi-Ma and @PDuwe)

___
*[`inspectit-ocelot-documentation/docs/enduser-monitoring/eum-server-configuration.md` line 483 at r1](https://reviewable.io/reviews/inspectIT/inspectit-ocelot/1569#-NMcLIJTOvLaoVSApx1h-r1-483:-NMcOuUx1kKDelfXScmJ:b-52dlrf) ([raw file](https://github.com/inspectIT/inspectit-ocelot/blob/59b08a083489eb329eaaa492fad46e03a75e7e15/inspectit-ocelot-documentation/docs/enduser-monitoring/eum-server-configuration.md#L483)):*
<details><summary><i>Previously, Dimi-Ma (Dimitiros Mantas) wrote…</i></summary><blockquote>

It might lead to misunderstandings if you say that this snippet shows the default configuration of the prometheus-exporter with it being enabled but say that it is disabled by default. 
</blockquote></details>

I agree.
this was a sentence "from the past", when Prometheus was enabled by default. I have adjusted it accordingly.

Thanks for catching this!

**Review by Dimi-Ma — CHANGES_REQUESTED** [review]
> Dismissed @PDuwe from a discussion.
> *[Reviewable](https://reviewable.io/reviews/inspectIT/inspectit-ocelot/1569)* status: 1 of 2 files reviewed, 1 unresolved discussion (waiting on @heiko-holz)

___
*[`inspectit-ocelot-documentation/docs/enduser-monitoring/eum-server-configuration.md` line 476 at r2](https://reviewable.io/reviews/inspectIT/inspectit-ocelot/1569#-NMcTBJaBvExpZdQohtE:-NMcTBJb36IDaRe5EQ-n:bv3wrib) ([raw file](https://github.com/inspectIT/inspectit-ocelot/blob/96a8a0d609c13ab023704034e5dc80421e0b6ec4/inspectit-ocelot-documentation/docs/enduser-monitoring/eum-server-configuration.md#L476)):*
> ```Markdown
> By default, the prometheus exporter is disabled.
> 
> The following configuration snippet shows makes the prometheus-exporter exposing the metrics on port `8888`:
> ```

This sentence feels wrong :thinking: I would change it to: "The following configuration snippet shows how the prometheus-exporter exposes the metrics on port `8888`:"

**Review by heiko-holz — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/inspectIT/inspectit-ocelot/1569)* status: 1 of 2 files reviewed, 1 unresolved discussion (waiting on @Dimi-Ma)

___
*[`inspectit-ocelot-documentation/docs/enduser-monitoring/eum-server-configuration.md` line 476 at r2](https://reviewable.io/reviews/inspectIT/inspectit-ocelot/1569#-NMcTBJaBvExpZdQohtE:-NMcUdIn4L9nG-1QJLgv:b-igsa7p) ([raw file](https://github.com/inspectIT/inspectit-ocelot/blob/96a8a0d609c13ab023704034e5dc80421e0b6ec4/inspectit-ocelot-documentation/docs/enduser-monitoring/eum-server-configuration.md#L476)):*
<details><summary><i>Previously, Dimi-Ma (Dimitiros Mantas) wrote…</i></summary><blockquote>

This sentence feels wrong :thinking: I would change it to: "The following configuration snippet shows how the prometheus-exporter exposes the metrics on port `8888`:"
</blockquote></details>

Yes, it was indeed grammatically wrong. I changed it.

**Review by Dimi-Ma — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r2, 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/inspectIT/inspectit-ocelot/1569#-:-NMcVUk13XCto5jb-wgE:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @heiko-holz)

**Pull request merged**